mempool/cat/pool.go

```diff
@@ -127,15 +129,11 @@ func WithMetrics(metrics *mempool.Metrics) TxPoolOption {
 	return func(txmp *TxPool) { txmp.metrics = metrics }
 }
 
-// Lock locks the mempool, no new transactions can be processed
-func (txmp *TxPool) Lock() {
-	txmp.mtx.Lock()
-}
+// Lock is a noop as ABCI calls are serialized
```

Comment:
what does this comment mean?

Reply:
I think it just means that that the lock doesn't serve a purpose, this just came from the revert